setup.py

```diff
@@ -16,7 +16,7 @@
 
 # Do not edit these constants. They will be updated automatically
 # by scripts/update-client.sh.
```

Comment:
i think scripts/update-client.sh needs to be updated too

Reply:
It looks like it uses constants.py to get the client version and update the setup.py. Am I missing something?

Reply:
ok. that's right.